### DISCUSSION TITLE
Web browser extension for recent problem solved

### DISCUSSION THREAD

**Discussion by PrakashGatiyala** [scaleracademy/open-source-projects#138] [Ideas]
We can create a web browser extension for recent problem solved on coding platform like codechef, codeforces, interviewbit and leetcode.  By enabling this extension, we will show the recent problem solved on these platform(Just like google recent history feature). 

- User can sort them by platform name(codechef, codeforces, interviewbit and leetcode).
- User can sort them using difficulty level. 
- User can bookmark their favorite Problems. 
- find the problem solved in specific time interval.

**Comment by HarshithSimha** [comment]
Great Idea, It will be helpful for many students.
